docs/advanced/server-side-rendering.md

```diff
@@ -177,254 +177,156 @@ we'll have to import from. `preact-ssr-prepass`.
 ## Next.js
 
 If you're using [Next.js](https://nextjs.org/) you can save yourself a lot of work by using
-`next-urql`. The `next-urql` package includes setup for `react-ssr-prepass` already, which automates
-a lot of the complexity of setting up server-side rendering with `urql`.
+`@urql/next`. The `@urql/next` package is set to work with Next 13.
 
-We have a custom integration with [`Next.js`](https://nextjs.org/), being [`next-urql`](https://github.com/urql-graphql/urql/tree/main/packages/next-urql)
-this integration contains convenience methods specifically for `Next.js`.
-These will simplify the above setup for SSR.
-
-To set up `next-urql`, first we'll install `next-urql` with `react-is` and `urql` as
+To set up `@urql/next`, first we'll install `@urql/next` and `urql` as
 peer dependencies:
 
 ```sh
-yarn add next-urql react-is urql graphql
+yarn add @urql/next urql graphql
 # or
-npm install --save next-urql react-is urql graphql
+npm install --save @urql/next urql graphql
 ```
 
-The peer dependency on `react-is` is inherited from `react-ssr-prepass` requiring it.
-
-Note that if you are using Next before v9.4 you'll need to polyfill fetch, this can be
-done through [`isomorphic-unfetch`](https://www.npmjs.com/package/isomorphic-unfetch).
+We now have two ways to leverage `@urql/next`, one being part of a Server component
+or being part of the general `app/` folder.
```

Comment:
For folks that are using Next 13, but haven't migrated to use the `app` folder yet, is the recommendation to continue using the `next-urql` package? or is `@urql/next` meant for those use cases as well?

Reply:
Apologies, just saw the PR description mentions:

> People can still use `next-urql` for the old next supported ways

I think it'd be good if these docs included mention of this. I also think the old docs (i.e. the ones for when using the [Pages Router](https://nextjs.org/docs/pages)) should probably stick around for the people who aren't able to upgrade to Next 13 and/or the app router yet. Perhaps this page ends up having two Next.js sections:
* Next.js (App Router)
* Next.js (Pages Router)

?

Reply:
This is a WIP, but eventually, I'm in favour of actually preserving the legacy docs in place, so people can still find the legacy package. So TBD